www/js/survey/enketo/enketo-add-note-button.js

```diff
@@ -60,40 +60,27 @@ angular.module('emission.survey.enketo.add-note-button',
     const momentBegin = begin ? moment(begin * 1000).tz(timezone) : null;
     const momentStop = stop ? moment(stop * 1000).tz(timezone) : null;
 
+    // the current, local time offset (e.g. -07:00)
+    const currOffset = moment().toISOString(true).slice(-6);
+
     const prefills = {}
     // Fill in only the fields that are present
     // Enketo requires these specific date/time formats
     if (momentBegin) {
       prefills.Start_date = momentBegin.format('YYYY-MM-DD');
-      prefills.Start_time = momentBegin.format('HH:mm:ss.SSS');
+      prefills.Start_time = momentBegin.format('HH:mm:ss.SSS') + currOffset;
```

Comment:
I am not 100% sure that this is correct. Shouldn't we add the offset before formatting as both date and time. What happens if the offset causes the date to change? How have you tested this?

Or if you want the original date and time, but just pretend to be in the current timezone, do you even need the currOffset? What happens if you don't set it?

Reply:
Date and time are stored in separate fields, so the time "overflowing" to another day isn't a worry here.
Note that I'm not adding time to the Moment object. I'm just concatenating a fake offset to the string that's going to Enketo.

> Or if you want the original date and time, but just pretend to be in the current timezone, do you even need the currOffset?

Yes; we have to pretend, for the purpose of the survey, that the trip times are in the current local time. Otherwise, Enketo will forcibly convert them to the local timezone.
If I don't put any offset (how it was before this change), the prefilled times display like we want them to, but it fails validation and the survey can't be submitted.

I tried everything I could think of and this is the only workaround, short of modifying Enketo again.